optics-core/src/Optics/IxSetter.hs

```diff
@@ -1,9 +1,9 @@
 {-# LANGUAGE DataKinds #-}
 -- |
 -- Module: Optics.IxSetter
--- Description: An indexed version of an 'Optics.Setter.Setter'.
+-- Description: An indexed version of a 'Optics.Setter.Setter'.
```

Comment:
Most of these article changes optimize for how it reads in the rendered haddock (which doesn't show the module name prefix). E.g.
![Screenshot from 2019-09-03 14-51-30](https://user-images.githubusercontent.com/2716069/64178538-ee7f9900-ce61-11e9-9ef1-ecabdee67c64.png)